### PR TITLE
[WebRTC] Change state restriction

### PR DESCRIPTION
**Pull request by hsgwon**
### Description of Change ###

Change API state restriction

### API Changes ###

 - TCSACR-585

### CODE REVIEW

File: src/Tizen.Multimedia.Remoting/WebRTC/MediaFileSource.cs [hunk 1]
```diff
@@ -107,7 +107,7 @@ public TransceiverDirection GetTransceiverDirection(MediaType type)
         /// Sets the transceiver direction for receiving media stream.
         /// </summary>
         /// <remarks>
-        /// The WebRTC must be in the <see cref="WebRTCState.Idle"/> state when transceiver direction is set.
+        /// This method does not throw state exception anymore(Since API12). It can be called in any state.<br/>
         /// </remarks>
         /// <param name="type">The media type.</param>
```
Comment (wiertel) [inline src/Tizen.Multimedia.Remoting/WebRTC/MediaFileSource.cs:112]: consider removing note at line 117 or adding an information that this changed in API 12

File: src/Tizen.Multimedia.Remoting/WebRTC/MediaNullSource.cs [hunk 2]
```diff
@@ -64,8 +64,7 @@ public TransceiverCodec GetTransceiverCodec(MediaType type)
         /// Sets the transceiver codec for receiving media stream.
         /// </summary>
         /// <remarks>
-        /// The WebRTC must be in the <see cref="WebRTCState.Idle"/> state when transceiver codec is set.<br/>
-        ///
+        /// This method does not throw state exception anymore(Since API12). It can be called in any state.<br/>
         /// </remarks>
         /// <param name="type">The media type.</param>
         /// <param name="codec">The transceiver codec.</param>
```
Comment (wiertel) [inline src/Tizen.Multimedia.Remoting/WebRTC/MediaNullSource.cs:70]: consider change in 74

File: src/Tizen.Multimedia.Remoting/WebRTC/WebRTC.cs [hunk 3]
```diff
@@ -378,7 +386,7 @@ public void AddIceCandidates(IEnumerable<string> iceCandidates)
         /// Adds media source.
         /// </summary>
         /// <remarks>
-        /// The WebRTC must be in the <see cref="WebRTCState.Idle"/>.<br/>
+        /// This method does not throw state exception anymore(Since API12). It can be called in any state.<br/>
         /// Each MediaSource requires different feature or privilege.<br/>
         /// <see cref="MediaCameraSource"/> needs camera feature and privilege.<br/>
         /// <see cref="MediaMicrophoneSource"/> needs microphone feature and recorder privilege.<br/>
```
Comment (wiertel) [inline src/Tizen.Multimedia.Remoting/WebRTC/WebRTC.cs:392]: line 404?

**Review by wiertel — CHANGES_REQUESTED** [review]

File: src/Tizen.Multimedia.Remoting/WebRTC/MediaFileSource.cs (hunk 1 above)
Comment (hsgwon) [reply src/Tizen.Multimedia.Remoting/WebRTC/MediaFileSource.cs:112]: I'll remove line 117.

File: src/Tizen.Multimedia.Remoting/WebRTC/MediaNullSource.cs (hunk 2 above)
Comment (hsgwon) [reply src/Tizen.Multimedia.Remoting/WebRTC/MediaNullSource.cs:70]: I'll remove line 74.

File: src/Tizen.Multimedia.Remoting/WebRTC/WebRTC.cs (hunk 3 above)
Comment (hsgwon) [reply src/Tizen.Multimedia.Remoting/WebRTC/WebRTC.cs:392]: I'll fix it.

**Review by wiertel — APPROVED** [review]

**Pull request merged**